crates/gen/src/types/class.rs

```diff
@@ -197,9 +195,28 @@ impl Class {
                         <::winrt::ComPtr<#default_name> as ::winrt::AbiTransferable>::set_abi(&mut self.ptr)
                     }
                 }
+                impl ::std::convert::From<#name> for ::winrt::Object {
```

Comment:
Is there a way to achieve this with a blanket impl? I wonder if having a marker trait for all WinRT classes and having a blanket impl `impl <T: Class> From<T> for Object` would be better for compile times.

Reply:
The trouble is that unlike in C++, Rust doesn't support negation, conjunction, or disjunction as generic constraints. Traits need to map to non-overlapping sets. I have been thinking about introducing non-overlapping marker traits for class/interface/delegate/struct so we can at least provide blanket implementations for subsets of WinRT types because so many of these are identical.